src/solvers/sweep_solve.jl

```diff
@@ -0,0 +1,40 @@
+
+default_region_callback(problem; kws...) = nothing
+
+default_sweep_callback(problem; kws...) = nothing
+
+function default_sweep_printer(problem; outputlevel, sweep, nsweeps, kws...)
```

Comment:
Similar to my comment above, what about calling these just `region_callback`, `sweep_callback`, and `sweep_printer`, since they would get overloaded on the `problem` type anyway?